jazz_codeq/components/config.js

```diff
@@ -0,0 +1,34 @@
+// =========================================================================
+// Copyright © 2017 T-Mobile USA, Inc.
+// 
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// =========================================================================
+
+const getStageConfig = (event) => {
+  const stage = event.stage;
+  // Loads the config files based on the env.
+  
+  const stageMap = {
+    'dev': '../config/dev-config.json',
+    'stg': '../config/stg-config.json',
+    'prod': '../config/prod-config.json'
+  };
+
+  const configObj = stageMap[stage] ? require(stageMap[stage]) : {}

```

Comment:
Then it will return the empty object as per original code - https://bitbucket.corporate.t-mobile.com/projects/CAP/repos/jazz_codeq/browse/components/config.js